### PR TITLE
V3 upload metadata android

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

**Review by syoung-smallwisdom — APPROVED** [review]
looks good?

File: assessmentmodel-sdk/src/main/java/org/sagebionetworks/bridge/assessmentmodel/upload/AssessmentResultArchiveUploader.kt [hunk 1]
```diff
@@ -90,7 +93,13 @@ abstract class AssessmentResultArchiveUploader(
             assessmentResult.runUUIDString
         }
 
-        val uploadFile = persist(assessmentRunUUID, builder.build(), sessionWindowExpiration)
+        val uploadMetadata: Map<String, JsonElement> = mapOf(
+            "instanceGuid" to JsonPrimitive(assessmentInstanceId),
+            "eventTimeStamp" to JsonPrimitive(eventTimestamp)
```
Comment (Erin-Mounts) [inline assessmentmodel-sdk/src/main/java/org/sagebionetworks/bridge/assessmentmodel/upload/AssessmentResultArchiveUploader.kt]:
```suggestion
            "eventTimestamp" to JsonPrimitive(eventTimestamp)
```
? Or does the key capitalization not matter (or did I get it wrong on iOS)?

**Review by Erin-Mounts — APPROVED** [review]
One question, otherwise looks good

File: assessmentmodel-sdk/src/main/java/org/sagebionetworks/bridge/assessmentmodel/upload/AssessmentResultArchiveUploader.kt (hunk 1 above)
Comment (nategbrown9) [reply assessmentmodel-sdk/src/main/java/org/sagebionetworks/bridge/assessmentmodel/upload/AssessmentResultArchiveUploader.kt]: Good catch. I thought I had just copied from iOS.

**Pull request merged**